Makefile

```diff
@@ -22,14 +22,72 @@ pg_test: install deps pg_build lint unittest unlink_brotli pg_integration_test
 pg_build: $(CMD_FILES) $(PKG_FILES)
 	(cd $(MAIN_PG_PATH) && go build -o wal-g $(GOTAGS) -ldflags "-s -w -X github.com/wal-g/wal-g/cmd.BuildDate=`date -u +%Y.%m.%d_%H:%M:%S` -X github.com/wal-g/wal-g/cmd.GitRevision=`git rev-parse --short HEAD` -X github.com/wal-g/wal-g/cmd.WalgVersion=`git tag -l --points-at HEAD`")
 
-pg_int_tests_only:
-	docker-compose build pg_tests
-	docker-compose up --exit-code-from pg_tests pg_tests
-
 pg_integration_test:
 	docker-compose build $(DOCKER_COMMON) pg pg_tests
 	docker-compose up --exit-code-from pg_tests pg_tests
 
+pg_unittests: install deps lint unittest
+
+pg_integration_delete_before_name_find_full_test: install deps pg_build unlink_brotli
```

Comment:
You can create a target with parameters in Makefile. So there won't be necessary to create all these similar targets.

Reply:
Done